### ISSUE TITLE
Why does Pykin prefix all filepaths with the site package path?

### ISSUE THREAD

**Issue by trzy** [jdj2261/pykin#10]
I wanted to define my own URDF file and was surprised to find this in urdf_model.py:28:

self.file_path = pykin_path + "/assets/" + f_name

This makes the package unusable from a pip install. Is there a reason for doing this?

**Comment by jdj2261** [comment]
Hello, instead of solving the robot's kinematics with a custom URDF, I decided to utilize a well-known URDF. 

I created a directory called "assets" in the form of a package. 

As you suggested, it would be wise to modify the code to test it with your own URDF. Thank you.

**Comment by trzy** [comment]
Yes I think that would make it more useful as a package (with pip install), otherwise it is a standalone project that is not easily reusable. Sent from my iPhoneOn Apr 15, 2024, at 4:16 AM, DaeJong Jin ***@***.***> wrote:﻿
Hello, instead of solving the robot's kinematics with a custom URDF, I decided to utilize a well-known URDF.
I created a directory called "assets" in the form of a package.
As you suggested, it would be wise to modify the code to test it with �your own URDF. Thank you.
